src/apps/integrations/oneup_health/service/task.py

```diff
@@ -98,21 +102,37 @@ async def _schedule_retry(
         unique_id (uuid.UUID): The unique identifier for the user
         start_date (datetime): The start date of the transfer process
         retry_count (int): The current retry attempt count
+        error_retry_count (int): The current error retry attempt count
     """
     delay = _exponential_backoff(retry_count)
     if delay > 0:
         retry_count += 1
         logger.info(f"Scheduling retry #{retry_count} in {delay} seconds")
-        await (
-            task_ingest_user_data.kicker()
-            .with_labels(delay=delay)
-            .kiq(applet_id=applet_id, unique_id=unique_id, start_date=start_date, retry_count=retry_count)
-        )
+        try:
+            await (
+                task_ingest_user_data.kicker()
+                .with_labels(delay=delay)
+                .kiq(applet_id=applet_id, unique_id=unique_id, start_date=start_date, retry_count=retry_count)
+            )
```

Comment:
@rcmerlo, @SebasYanik I know it wasn't specified in the AC, but should we consider the max number of attempts after _n_ **consecutive** failures (ie. 1up is down for a single extended period). If so then maybe we should reset after a successful retry

```suggestion
            await (
                task_ingest_user_data.kicker()
                .with_labels(delay=delay)
                .kiq(applet_id=applet_id, unique_id=unique_id, start_date=start_date, retry_count=retry_count)
            )
            error_retry_count = 0
```

Reply:
I agree we should reset error count when we get a successfull response, even if ingestion is not completed yet